GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1167,6 +1176,10 @@ private void UpdatePlaceholders()
                     }
                 }
 
+                stopwatch.Stop();
+                long millisecondsRemovingEmptyFolders = stopwatch.ElapsedMilliseconds;

```

Comment:
Should we also include a count of how many folders were successfully deleted? (I'm not sure if this should go in `GVFSLock.Stats` though or be recorded to telemetry somewhere else.